packages/types/app/index.d.ts

```diff
@@ -100,3 +100,8 @@ export interface NuxtApp extends Vue {
   refresh(): void
   setLayout(layout: string): any // TBD
 }
+
+// Global Nuxt Object accessible outside components
+declare global {
+  const $nuxt: NuxtApp
```

Comment:
I think that it should not be declared as variable but more likely as the correct property of `window` (`window.$nuxt`), so directly around the global `Window` interface from DOM types.

```suggestion
  interface Window {
    $nuxt: NuxtApp
  }
```

Reply:
@kevinmarrec Have you tested it? I have build error with your solution :(

Reply:
Yes. You should use `window.$nuxt` instead of just `$nuxt` in your application.

Reply:
![image](https://user-images.githubusercontent.com/25272043/69875933-8f91ba80-12be-11ea-8ee0-3a613ef94e11.png)